src/Microsoft.AspNet.SignalR.Client.UWP/Microsoft.AspNet.SignalR.Client.UWP.csproj

```diff
@@ -11,11 +11,12 @@
     <AssemblyName>Microsoft.AspNet.SignalR.Client</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.10586.0</TargetPlatformVersion>
+    <TargetPlatformVersion>10.0.17134.0</TargetPlatformVersion>
     <TargetPlatformMinVersion>10.0.10240.0</TargetPlatformMinVersion>
     <MinimumVisualStudioVersion>14</MinimumVisualStudioVersion>
     <FileAlignment>512</FileAlignment>
     <ProjectTypeGuids>{A5A43C5B-DE2A-4C0C-9213-0A381AF9435A};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+    <RuntimeIdentifiers>win10-arm;win10-arm-aot;win10-x86;win10-x86-aot;win10-x64;win10-x64-aot</RuntimeIdentifiers>
```

Comment:
Seems like an odd PR to upgrade these in.

Reply:
Why is this even here?

Reply:
I can revert it all, but it means people have to install older Windows 10 SDKs (rather than the latest that comes with the latest VS updates) to open the project in VS 2017.